management-api-server/src/main/java/com/datastax/mgmtapi/resources/MetadataResources.java

```diff
@@ -102,4 +102,20 @@ public Response getFeatureSet() {
           return Response.ok(featureSet).build();
         });
   }
+
+  @GET
+  @Path("/clustername")
+  @Operation(summary = "Returns the Cassandra cluster name", operationId = "getClusterName")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Cassandra Cluster Name",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
```

Comment:
Isssue: Can we have this wrapped in JSON please :)

Reply:
For this one, there isn't a need since a simple String will suffice. But actually this may be moot for now because I like the idea of sticking this information in the endpoint states as it would mean no new endpoint.